src/ontology/edits/oeo-physical.omn

```diff
@@ -3733,6 +3737,248 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/768",
+Class: OEO_00010104
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A water body is an accumulation of liquid water of varying size on the surface of the Earth.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/762
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/806",
+        rdfs:label "water body"@en
+    
+    SubClassOf: 
+        OEO_00110000
+    
+    
+Class: OEO_00010105
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "The ocean is a water body that consists of salt water and is covering approximately 71% of Earth's surface.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "marine water body",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "sea",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/762
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/806",
+        rdfs:label "ocean"@en
+    
+    SubClassOf: 
+        OEO_00010104
+    
+    
+Class: OEO_00010106
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A wave is a water flow that is at the surface of a water body/marine water body/ocean and that is mainly vertically orientated.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/762
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/806",
+        rdfs:label "wave"@en
+    
+    SubClassOf: 
+        OEO_00110002
+    
+    
+Class: OEO_00010107
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An ocean current is a a water flow within a water body/marine water body/ocean that is mainly horizontally orientated.",
+        rdfs:label "ocean current"@en
+    
+    SubClassOf: 
+        OEO_00110002
+    
+    
+Class: OEO_00010108
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A marine current energy converting unit is a hydro power unit that uses marine current energy.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/762
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/806",
+        rdfs:label "marine current energy converting unit"@en
+    
+    SubClassOf: 
+        OEO_00010085
+    
+    
+Class: OEO_00010109
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A marine tidal energy converting unit is a hydro power unit that uses marine tidal energy.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/762
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/806",
+        rdfs:label "marine tidal energy converting unit"@en
+    
+    SubClassOf: 
+        OEO_00010085,
+        <http://purl.obolibrary.org/obo/RO_0000056> some OEO_00010098
```

Comment:
This should be `participates in some marine tidal energy transformation` (instead of marine current).